lib/process-services-cloud/src/lib/process/start-process/services/start-process-cloud.service.ts

```diff
@@ -63,9 +62,7 @@ export class StartProcessCloudService extends BaseCloudService {
         const url = `${this.getBasePath(appName)}/rb/v1/process-instances`;
         payload.payloadType = 'StartProcessPayload';
 
-        return this.post(url, payload).pipe(
-            map((result: any) => result.entry)
-        );
+        return this.post(url, payload).pipe(map((result: any) => result.entry));
```

Comment:
Okay I would love to see those `any` gone but probably in other ticket since the further I go the more I see :D maybe let's stick to `any` that you've added when emitting errors